rails_app/app/jobs/deals_scrape_job.rb

```diff
@@ -0,0 +1,35 @@
+class DealsScrapeJob < ApplicationJob
+  queue_as :default
+
+  @@graphite = GraphiteAdapter.new
```

Comment:
its not the best solution, but for now let it be like that

Reply:
and why is that?